src/storage/txn/scheduler.rs

```diff
@@ -48,10 +50,13 @@ use super::Result;
 use super::Error;
 use super::store::SnapshotStore;
 use super::latch::{Latches, Lock};
+use super::statistics::{RegionsWriteStats, ROLL_INTERVAL_SECS};
 use super::super::metrics::*;
 
 // TODO: make it configurable.
 pub const GC_BATCH_SIZE: usize = 512;
+pub const INIT_WRITE_STATS: u64 = 16;
+const U8_MAX: u64 = 255;
```

Comment:
You can use `u8::max_value()` instead. https://doc.rust-lang.org/stable/std/primitive.u8.html#method.max_value